staging/src/k8s.io/component-base/metrics/testutil/metrics.go

```diff
@@ -19,11 +19,15 @@ package testutil
 import (
 	"fmt"
 	"io"
+	"math"
 	"reflect"
+	"sort"
 	"strings"
 
+	dto "github.com/prometheus/client_model/go"
 	"github.com/prometheus/common/expfmt"
 	"github.com/prometheus/common/model"
+	"k8s.io/component-base/metrics/legacyregistry"
```

Comment:
Could you please move `k8s.io/xxx` to a separate group?

Reply:
done

Reply:
We shouldn't import the legacyregistry into testutil, eventually we want to purge that functionality and this will make things a bit more complicated.